www/script/project/rtBuilders.js

```diff
@@ -165,19 +129,23 @@ define(function (require) {
                         branch_tags.add(tag.toLowerCase());
                     }
                 });
+
+                if (builderTags.length > 1){
+                    tags.add(builderTags.join(tagSeparator));
```

Comment:
So this is an alternative implementation of AND condition, now list of tags contains not only unique tag lists - but unique existing combinations.